data/tips.cfg

```diff
@@ -1,15 +1,15 @@
 #textdomain wesnoth
 
 [tip]
-    text= _ "Use the map editor's <i>Areas</i> menu to define regions later addressed by event or AI coding."
+    text= _ "Use the map editor’s <i>Areas</i> menu to define regions later addressed by event or AI coding."
     source= _ "<i>― The Wesnoth Design Guide</i>"
 [/tip]
 [tip]
-    text= _ "Create a <i>New Scenario</i> in the editor's <i>File</i> menu to make use of advanced features."
+    text= _ "In the map editor, create a <i>New Scenario</i> in the <i>File</i> menu to make use of advanced features."
     source= _ "<i>― The Wesnoth Design Guide</i>"
 [/tip]
 [tip]
-    text= _ "Scenarios saved from the editor in their default folder can be selected in the <i>User Scenarios</i> section of the Multiplayer ""Create Game"" screen."
+    text= _ "Scenarios saved from the editor in their default folder can be selected in the <i>Custom Scenarios</i> section of the Multiplayer ‘Create Game’ screen."
```

Comment:
Good catch on the first correction. As to the second, the double double quotes are not an error (see [SyntaxWML](https://wiki.wesnoth.org/SyntaxWML#Special_Attribute_Values)); is there a particular reason to use single quotes here?

Reply:
I misunderstood the [Typography Style Guide](https://wiki.wesnoth.org/Typography_Style_Guide), missing that this is an exact quote from the create game screen.